docs/release/new-release-series.md

```diff
@@ -121,23 +116,24 @@ Do on the month of the 3.4.0 release
 - Update documentation [here](/software/development-process)
 - Update osg-test / vmu-test-runs
     -   They're only going to test from minefield (and eventurally testing) until the release
-- Drop the `osg-3.4-elX-bootstrap` koji tags (after the successful mass rebuild only)
+- Drop the `osg-3.X-elY-bootstrap` koji tags (after the successful mass rebuild only)
```

Comment:
Could you add a bullet here mentioning that we should update https://github.com/opensciencegrid/docker-software-base and any container images that are based on it?

Reply:
updated